components/brave_wallet/browser/permission_utils_unittest.cc

```diff
@@ -51,30 +45,26 @@ TEST(PermissionUtilsUnitTest, GetConcatOriginFromWalletAddresses) {
       }};
 
   url::Origin origin = url::Origin::Create(GURL("https://test.com"));
-  for (size_t i = 0; i < sizeof(cases) / sizeof(cases[0]); ++i) {
+  for (const auto& test_case : cases) {
+    SCOPED_TRACE(testing::Message() << test_case.expected_out_origin);
+
     url::Origin out_origin;
-    EXPECT_TRUE(
-        GetConcatOriginFromWalletAddresses(origin, cases[i].addrs, &out_origin))
-        << "case: " << i;
+    EXPECT_TRUE(GetConcatOriginFromWalletAddresses(origin, test_case.addrs,
+                                                   &out_origin));
```

Comment:
These tests don't seem to be checking the out param. I've made a not that `GetConcatOriginFromWalletAddresses` should be returning an `optional<url::Origin>`, rather than using an outparam, but this can be handled on a follow up.

Reply:
https://github.com/brave/brave-core/pull/26431/commits/6809115eb5276f7e5f97305fc0e0189ade29d515

Reply:
As I mentioned above this should have been done on a follow up, as this PR is already large as it is.

Reply:
You said "can be" not "should be".